Airports/EGVH/Basic.txt

```diff
@@ -0,0 +1,3 @@
+Hereford
+N052.04.51.08 W002.47.56.39
+000
```

Comment:
```suggestion
000.000
```
Genuinely no idea if this should be the frequency of Gloster Radar (128.555 or 120.980), some other MIL freq, or 000.000 for not covered - I suggest we leave it at `000.000` until someone gets back to me in `#sector-file-development`

Reply:
Been resolved - it's not top-down, so 000.000 will do.